contracts/base/LoansBase.sol

```diff
@@ -586,7 +582,8 @@ contract LoansBase is LoansInterface, Base {
         TellerCommon.LoanStatus currentStatus = loans[loanID].status;
         uint256 loanAmount;
         if (currentStatus == TellerCommon.LoanStatus.TermsSet) {
-            loanAmount = loans[loanID].loanTerms.maxLoanAmount.add(loans[loanID].interestOwed);
+            uint256 interestOwed = _getInterestOwed(loanID, loans[loanID].loanTerms.maxLoanAmount);
+            loanAmount = loans[loanID].loanTerms.maxLoanAmount.add(interestOwed);
         } else if (currentStatus == TellerCommon.LoanStatus.Active) {
             loanAmount = loans[loanID].interestOwed.add(loans[loanID].principalOwed);
```

Comment:
Just a preference to have the principalOwed before the interest 😅

Reply:
Got it, changed